### PR TITLE
Doc: Replace heroku service with postman in tutorial part 1

### LINKED ISSUE #875: Heroku Service for consumer testing down?
Hello!

I am going through the tutorials on setting up Oauth Toolkit on readthedocs,
there is a section on the tutorial for testing the authorization servers using a heroku consumer service if I am not very familiar with Oauth2 (I am definitely not familiar with oauth2 heheh), but the site for testing that authorization is down due to an application error.
Is there someone I should contact to get that fixed or up and running?

### PR DESCRIPTION
**Pull request by marcusl**
Fixes #875 

## Description of the Change

Rewrite documentation to use postman and not the defunct Heroku service.

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [N/A] unit-test added
- [x] documentation updated
- [N/A] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: docs/tutorial/tutorial_01.rst [hunk 1]
```diff
@@ -105,17 +105,28 @@ process we'll explain shortly)
 Test Your Authorization Server
 ------------------------------
 Your authorization server is ready and can begin issuing access tokens. To test the process you need an OAuth2
-consumer; if you are familiar enough with OAuth2, you can use curl, requests, or anything that speaks http. For the rest
-of us, there is a `consumer service <http://django-oauth-toolkit.herokuapp.com/consumer/>`_ deployed on Heroku to test
-your provider.
+consumer; if you are familiar enough with OAuth2, you can use curl, requests, or anything that speaks http.
+
+For this tutorial, we suggest using Postman, where we will be following `this guide https://columbia-it-django-jsonapi-training.readthedocs.io/en/latest/using_oauth2/#get-an-oauth-20-token`.
```
Comment (n2ygk) [inline docs/tutorial/tutorial_01.rst:110]: I'm not sure if this doc should be linked. I'd prefer that this tutorial doc stand on it's own.
```suggestion
For this tutorial, we suggest using Postman.
```

**Review by n2ygk — CHANGES_REQUESTED** [review]
Thanks for this. Please make sure to add your name to AUTHORS as in the checklist as well as this suggested change.

**Comment by marcusl** [comment]
Done. :)

**Review by n2ygk — APPROVED** [review]
Thanks!

**Pull request merged**